src/foam/nanos/export/ExportDriver.js

```diff
@@ -24,6 +24,12 @@ foam.INTERFACE({
       args: ['X', 'dao'],
       documentation: 'Exports data in a DAO',
       flags: ['js']
+    },
+    {
+      name: 'cleanup',
```

Comment:
Often this would be called 'tearDown' (in testing frameworks) as it is the opposite of 'setUp', should there be one. Unless you are really tied to 'cleanup'.

Reply:
good to know! thanks.
fixed